test/integration/BUILD

```diff
@@ -61,7 +61,7 @@ py_library(
     deps = [":integration_test_base"],
 )
 
-py_test(
+py_binary(
```

Comment:
So py_test doesn't count for coverage but coverage works over forking.  OK.

Reply:
That was the plan, make python run the with instrumented binaries.. 
I thought that worked, however, double checking again this actually didn't work as intended.
So I just pushed another commit which declares some of the top-level libs as dependencies.
With that I could verify this to work.

Reply:
Did it work?

Reply:
Yes, I could verify that as of the last commit, it worked